governance/votes/Executive vote - July 29, 2022.md

```diff
@@ -0,0 +1,199 @@
+* **RWA008_A_INPUT_CONDUIT:** `0xa397a23dDA051186F202C67148c90683c413383C`.
+* **RWA008_A_OUTPUT_CONDUIT:** `0x21CF5Ad1311788D762f9035829f81B9f54610F0C`.
+* **RWA008_A_OPERATOR:** `0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439`.
+* **RWA008_A_MATE:** `0xb9444802F0831A3EB9f90E24EFe5FfA20138d684`.
+
+*Please note that **RWA008_A_OPERATOR** represents Societe Generale – Forge and **RWA008_A_MATE** represents DIIS group.*
+
+#### Additional Information
+
+Please review the following forum threads containing information about RWA-008 (OFH) to inform your position before voting.
+* [Initial Collateral Application](https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605)
+* [Real-World Finance Core Unit Risk Evaluation](https://forum.makerdao.com/t/sg-forge-socgen-risk-assessment/15638)
+* [Additional Risk Assessment - LOVE](https://forum.makerdao.com/t/sg-forge-socgen-risk-assessment/15638/14)
+* [Collateral Engineering Services Core Unit Evaluation](https://forum.makerdao.com/t/rwa008-ofh-mip21-token-ces-domain-team-assessment/15239)
+* [Oracles Core Unit Evaluation](https://forum.makerdao.com/t/rwa-008-ofh-collateral-onboarding-oracle-assessment/15268)
+* [SocGen Transaction Documentation](https://api.ipfsbrowser.com/ipfs/get.php?hash=QmW1UBVjPtHDo42oNzG2vspwggyeBfENS1xsPtpKLyDBvb)
+* [Delegated Committee Proposal](https://forum.makerdao.com/t/real-world-finance-decision-making-and-delegated-committee-proposal/15276)
+
+
+### RWA-009 Huntingdon Valley Bank (HVBank) Onboarding
+
+As per [this poll](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), RWA-009 Huntingdon Valley Bank (HVBank) will be onboarded with the following parameters, if this executive proposal passes.
+ 
+#### Collateral Parameters
+
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): 0.0%
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): 100 million DAI
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): 100%
+* Liquidation Process: See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+
+#### Relevant Smart Contract Information
+
+* **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`.
+* **RWA_TOKEN_FAB:** `0x2B3a4c18705e99bC29b22222dA7E10b643658552`.
+* **SYMBOL:** `RWA009`.
+* **NAME:** `RWA-009`.
+* **ILK:** `RWA009-A`.
+* **RWA009:** `0x8b9734bbaA628bFC0c9f323ba08Ed184e5b88Da2`.
+* **MCD_JOIN_RWA009_A:** `0xEe0FC514280f09083a32AE906cCbD2FAc4c680FA`.
+* **RWA009_A_URN:** `0x1818EE501cd28e01E058E7C283E178E9e04a1e79`.
+* **RWA009_A_JAR:** `0x6C6d4Be2223B5d202263515351034861dD9aFdb6`.
+* **RWA009_A_OUTPUT_CONDUIT:** `0x508D982e13263Fc8e1b5A4E6bf59b335202e36b4`.
+
+*Please note that:*
+- **RWA009_A_OUTPUT_CONDUIT** represents Genesis trading address (receivers of DAI who will swap to fiat).
```

Comment:
Can you add that the address can be verified here: https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/10?u=miguel_cabeza